modules/about-cli-profiles-switch.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_cli/managing-cli-profiles.adoc
+
+[id="about-switches-between-cli-profiles_{context}"]
+= About switches between CLI profiles
+
+Contexts allow you to easily switch between multiple users across multiple
+{product-title} servers, or clusters, when using CLI operations. Nicknames
+make managing CLI configurations easier by providing short-hand references to
+contexts, user credentials, and cluster details.
+
+After logging in with the CLI for the first time, {product-title} creates a
+`_~/.kube/config_` file if one does not already exist. As more authentication
+and connection details are provided to the CLI, either automatically during an
+`oc login` operation or by manually configuring CLI profiles, the updated
+information is stored in the configuration file:
+
+.CLI config file
+
+[source,yaml]
+----
+apiVersion: v1
+clusters: <1>
+- cluster:
+    insecure-skip-tls-verify: true
+    server: https://openshift1.example.com:8443
+  name: openshift1.example.com:8443
+- cluster:
+    insecure-skip-tls-verify: true
+    server: https://openshift2.example.com:8443
+  name: openshift2.example.com:8443
+contexts: <2>
+- context:
+    cluster: openshift1.example.com:8443
+    namespace: alice-project
+    user: alice/openshift1.example.com:8443
+  name: alice-project/openshift1.example.com:8443/alice
+- context:
+    cluster: openshift1.example.com:8443
+    namespace: joe-project
+    user: alice/openshift1.example.com:8443
+  name: joe-project/openshift1/alice
+current-context: joe-project/openshift1.example.com:8443/alice <3>
+kind: Config
+preferences: {}
+users: <4>
+- name: alice/openshift1.example.com:8443
+  user:
+    token: xZHd2piv5_9vQrg-SKXRJ2Dsl9SceNJdhNTljEKTb8k
+----
+
+<1> The `*clusters*` section defines connection details for {product-title} clusters,
+including the address for their master server. In this example, one cluster is
+nicknamed *openshift1.example.com:8443* and another is nicknamed
```

Comment:
`openshift1.example.com:8443` and another is nicknamed `openshift2.example.com:8443`